Detectors/ZDC/calib/testWorkflow/ZDCDCSDataProcessorSpec.h

```diff
@@ -0,0 +1,224 @@
+                                          "ZDC_ZPC_POS.actual.position",
+                                          "ZDC_ZNA_HV0.actual.vMon",
+                                          "ZDC_ZNA_HV1.actual.vMon",
+                                          "ZDC_ZNA_HV2.actual.vMon",
+                                          "ZDC_ZNA_HV3.actual.vMon",
+                                          "ZDC_ZNA_HV4.actual.vMon",
+                                          "ZDC_ZPA_HV0.actual.vMon",
+                                          "ZDC_ZPA_HV1.actual.vMon",
+                                          "ZDC_ZPA_HV2.actual.vMon",
+                                          "ZDC_ZPA_HV3.actual.vMon",
+                                          "ZDC_ZPA_HV4.actual.vMon",
+                                          "ZDC_ZNC_HV0.actual.vMon",
+                                          "ZDC_ZNC_HV1.actual.vMon",
+                                          "ZDC_ZNC_HV2.actual.vMon",
+                                          "ZDC_ZNC_HV3.actual.vMon",
+                                          "ZDC_ZNC_HV4.actual.vMon",
+                                          "ZDC_ZPC_HV0.actual.vMon",
+                                          "ZDC_ZPC_HV1.actual.vMon",
+                                          "ZDC_ZPC_HV2.actual.vMon",
+                                          "ZDC_ZPC_HV3.actual.vMon",
+                                          "ZDC_ZPC_HV4.actual.vMon",
+                                          "ZDC_ZEM_HV0.actual.vMon",
+                                          "ZDC_ZEM_HV1.actual.vMon",
+                                          "ZDC_ZNA_HV0_D[1..2]",
+                                          "ZDC_ZNC_HV0_D[1..2]",
+                                          "ZDC_ZPA_HV0_D[1..2]",
+                                          "ZDC_ZPC_HV0_D[1..2]"};
+      std::vector<std::string> aliasesInt = {"ZDC_CONFIG_[00..32]"};
+      std::vector<std::string> expaliases = o2::dcs::expandAliases(aliases);
+      std::vector<std::string> expaliasesInt = o2::dcs::expandAliases(aliasesInt);
+      for (const auto& i : expaliases) {
+        vect.emplace_back(i, o2::dcs::RAW_DOUBLE);
+      }
+    }
+
+    LOG(info) << "Listing Data Points for ZDC:";
+    for (auto& i : vect) {
+      LOG(info) << i;
+    }
+
+    mProcessor = std::make_unique<o2::zdc::ZDCDCSProcessor>();
+    bool useVerboseMode = ic.options().get<bool>("use-verbose-mode");
+    LOG(info) << " ************************* Verbose?" << useVerboseMode;
+    if (useVerboseMode) {
+      mProcessor->useVerboseMode();
+    }
+    mProcessor->init(vect);
+    mTimer = HighResClock::now();
+  }
+
+  void run(o2::framework::ProcessingContext& pc) final
+  {
+    auto startValidity = DataRefUtils::getHeader<DataProcessingHeader*>(pc.inputs().getFirstValid(true))->creation;
+    auto dps = pc.inputs().get<gsl::span<DPCOM>>("input");
+    auto timeNow = HighResClock::now();
```

Comment:
std::chrono::system_clock

Reply:
I think I changed what you asked for (I am pushing those right now). I did some test, using o2-calibration-zdc-dcs-sim-workflow, and it seemed to work. Unfortunately I could not use the DCS server emulator since I am having troubles in compiling the emulator...